msfvenom

```diff
@@ -69,81 +69,69 @@ def parse_args(args)
   opt.separator('')
   opt.separator('Options:')
 
+  opt.on('-l', '--list            <type>', Array, 'List all module for [type]. Types are: payloads, encoders, nops, platforms, encrypt, formats, all') do |l|
```

Comment:
I think it should be "list all modules."

Reply:
I agree